_sections/91-encrypted-forms.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Encryption 
+---
+Encrypted forms provide a mechanism to keep your data private even when using http: for communications (e.g., when you do not have an SSL certificate or https: is not available).
+
+### Form Definition
+
+Here is an excerpt used in an encrypted form:
+
+{% highlight xml %}
+<submission method="form-data-post"
```

Comment:
add the primary instance as well to show where the submission element goes

Reply:
Good idea. I suggested only including the `submission` block but it's true that it's not clear where it goes. I think adding 
```
<instance>
...
</instance>
```
before it would be sufficient.